apps/condo/domains/resident/schema/RegisterResidentService.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Generated by `createservice property.RegisterResidentService --type mutations`
+ */
+
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('../access/RegisterResidentService')
+const { Resident } = require('../utils/serverSchema/index')
+const { Property } = require('@condo/domains/property/utils/serverSchema')
+
+
+const RegisterResidentService = new GQLCustomSchema('RegisterResidentService', {
+    types: [
+        {
+            access: true,
+            type: 'input RegisterResidentInput { dv: Int!, sender: JSON!, address: String!, addressMeta: JSON!, unitName: String! }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.canRegisterResident,
+            schema: 'registerResident(data: RegisterResidentInput!): Resident',
+            resolver: async (parent, args, context, info, extra = {}) => {
+                if (!context.authedItem.id) throw new Error('[error] User is not authenticated')
+                const { data: { dv, sender, address, addressMeta, unitName } } = args
```

Comment:
add validation

Reply:
If You mean validation of `addressMeta`, look at [my comment](https://github.com/open-condo-software/condo/pull/313#discussion_r677567846) please.
Validation of another fields will be performed by `Property` and `Resident` schemas.